test/data/extensions/test_publisher_proof_key_alt.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
It's just a test key, isn't?
Just to be sure we will not use it in production.

Reply:
Yes, this is just a test key. We do not use it in prod.